proto/experiment/service.proto

```diff
@@ -293,6 +308,40 @@ service ExperimentService {
       }
     };
   }
+  rpc GetGoalConnectionReport(GetGoalConnectionReportRequest)
+      returns (GetGoalConnectionReportResponse) {
+    option (google.api.http) = {
+      get: "/v1/goal/connection-report"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Get connection report"
+      description: "Get goal connection report."
```

Comment:
This API returns a summary of the total number of experiments, auto operations, and not-in-use goals for the last and current months.